Command/CacheCommandTrait.php

```diff
@@ -60,7 +60,7 @@ private function setupOutputStyle(InputInterface $input, OutputInterface $output
     private function resolveInputFiltersAndPaths(InputInterface $input): array
     {
         return [
-            $input->getArgument('path'),
+            $input->getArgument('paths'),
```

Comment:
is this a bugfix? if not, this could be a BC break

Reply:
ah sorry, its an argument, not an option! so no name on the cli that the user of the command types.